plugin.rb

```diff
@@ -225,6 +225,11 @@ class DiscourseEncrypt::Engine < Rails::Engine
   on(:reduce_cooked) do |fragment, post|
     if post&.is_encrypted?
       fragment.inner_html = "<p>#{I18n.t('js.encrypt.encrypted_post_email')}</p>"
+      timer = post.encrypted_post_timer || post.topic.posts.first.encrypted_post_timer
+      if timer
```

Comment:
Super trivial, but one nice thing you can do in ruby is say `if timer = (post.encrypted_post_timer || post.topic.posts.first.encrypted_post_timer)` - saves you a line!